charts/neuvector/latest/templates/pvc.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if and .Values.controller.enabled .Values.controller.pvc.enabled -}}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: neuvector-data
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: {{ template "neuvector.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  accessModes:
+    - ReadWriteMany
+  volumeMode: Filesystem
+{{- if .Values.controller.pvc.storageClass }}
+  storageClassName: {{ .Values.controller.pvc.storageClass }}
+{{- end }}
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
Same for the storage size, this should be configurable.

Reply:
Usually, 1G size is enough as we don't use PVC to store for large amount of data, but definitely it's more user friendly to make it configurable. We can enable that in the future updates.